examples/mlmc.cpp

```diff
@@ -160,6 +143,19 @@ int main(int argc, char* argv[])
     bool coarse_components = true;
     args.AddOption(&coarse_components, "-coarse-comp", "--coarse-components", "-no-coarse-comp",
                    "--no-coarse-components", "Store trace, bubble components of coarse M.");
+    const char* sampler_type = "simple";
+    args.AddOption(&sampler_type, "--sampler-type", "--sampler-type",
+                   "Which sampler to use for coefficient: simple, pde");
+    double kappa = 0.001;
+    args.AddOption(&kappa, "--kappa", "--kappa",
+                   "Correlation length for Gaussian samples.");
+    int num_samples = 3;
+    args.AddOption(&num_samples, "--num-samples", "--num-samples",
+                   "Number of samples to draw and simulate.");
+    bool blank_perm = false;
```

Comment:
would it be simpler to have the default empty filename string imply blank permutation?  That way all the complexity is moved into `IPF::ReadPermeabilityFile`. Everything else would be unchanged: there would be the single spe10 constructor the and the spe10 object would not need to be a unique pointer.